### PR TITLE
IDEMPIERE-2955: Adding Attribute Set Type to use attribute set for di…

### PR DESCRIPTION
**Pull request by dpansheriya**
…fferent purpose

@globalqss as discussed on IDEMPIERE-4224, This ticket move us one step towards de-cluttering configuration implementation.

### CODE REVIEW

File: migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-- Dec 2, 2015 11:56:12 AM IST
+INSERT INTO AD_Ref_List (AD_Ref_List_ID,Name,AD_Reference_ID,Value,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,AD_Ref_List_UU) VALUES (200344,'Material Management System',200115,'MMS',0,0,'Y',TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,'D','4455f892-2ea1-499d-a0b9-c01b3226f97d')
+;
+
+-- Dec 2, 2015 11:56:29 AM IST
+UPDATE AD_Column SET AD_Reference_Value_ID=200115,Updated=TO_TIMESTAMP('2015-12-02 11:56:29','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212554
+;
+
+-- Dec 2, 2015 11:56:42 AM IST
+ALTER TABLE M_AttributeSet ADD COLUMN M_AttributeSet_Type VARCHAR2(3 CHAR) DEFAULT 'MMS'
```
Comment (globalqss) [inline migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql:26]: this statement throws error in oracle:
ORA-00904: : invalid identifier

File: migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+-- Dec 2, 2015 11:54:23 AM IST
+INSERT INTO AD_Reference (AD_Reference_ID,Name,ValidationType,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,IsOrderByValue,AD_Reference_UU) VALUES (200115,'AttributeSet Type','L',0,0,'Y',TO_TIMESTAMP('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,'D','N','dc52c0cb-adec-4c9e-a176-29ca09f0072e')
+;
+
+-- Dec 2, 2015 11:56:12 AM IST
+INSERT INTO AD_Ref_List (AD_Ref_List_ID,Name,AD_Reference_ID,Value,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,AD_Ref_List_UU) VALUES (200344,'Material Management System',200115,'MMS',0,0,'Y',TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,'D','4455f892-2ea1-499d-a0b9-c01b3226f97d')
+;
+
+-- Dec 2, 2015 11:56:29 AM IST
+UPDATE AD_Column SET AD_Reference_Value_ID=200115,Updated=TO_TIMESTAMP('2015-12-02 11:56:29','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212554
```
Comment (globalqss) [inline migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql:22]: There are 9 TO_TIMESTAMP here and above - the correct syntax in oracle is TO_DATE

File: migration/i7.1z/postgresql/201602161510_IDEMPIERE-2955.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+-- Dec 2, 2015 11:54:23 AM IST
+INSERT INTO AD_Reference (AD_Reference_ID,Name,ValidationType,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,IsOrderByValue,AD_Reference_UU) VALUES (200115,'AttributeSet Type','L',0,0,'Y',TO_DATE('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2015-12-02 11:54:22','YYYY-MM-DD HH24:MI:SS'),100,'D','N','dc52c0cb-adec-4c9e-a176-29ca09f0072e')
+;
+
+-- Dec 2, 2015 11:56:12 AM IST
+INSERT INTO AD_Ref_List (AD_Ref_List_ID,Name,AD_Reference_ID,Value,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,AD_Ref_List_UU) VALUES (200344,'Material Management System',200115,'MMS',0,0,'Y',TO_DATE('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2015-12-02 11:56:10','YYYY-MM-DD HH24:MI:SS'),100,'D','4455f892-2ea1-499d-a0b9-c01b3226f97d')
+;
+
+-- Dec 2, 2015 11:56:29 AM IST
+UPDATE AD_Column SET AD_Reference_Value_ID=200115,Updated=TO_DATE('2015-12-02 11:56:29','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212554
```
Comment (globalqss) [inline migration/i7.1z/postgresql/201602161510_IDEMPIERE-2955.sql:19]: There are 9 TO_DATE here and above - the correct syntax in postgresql is TO_TIMESTAMP

**Review by globalqss — CHANGES_REQUESTED** [review]
Please fix migration scripts

**Comment by dpansheriya** [comment]
@globalqss The migration script are swept between oracle and postgresql folders. Issue fixed.

File: migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql [hunk 4]
```diff
@@ -0,0 +1,50 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- IDEMPIERE-2955 Adding Attribute set type field (M_AttributeSet_Type), support DMS/MM
+ Dec 2, 2015 11:53:13 AM IST
```
Comment (globalqss) [inline migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql:5]: error:
SP2-0734: unknown command beginning "Dec 2, 201..." - rest of line ignored.

**Review by globalqss — CHANGES_REQUESTED** [review]

File: migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql (hunk 4 above)
Comment (dpansheriya) [reply migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql:5]: @globalqss No oracle DB server, so will install, test script and then do next commit. This way is too costly:)

Comment (globalqss) [reply migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql:5]: I'm testing locally in an docker oracle - this seems the last one - the line starting with "Dec 2" must be preceded by --

Probably you deleted the comment mark inadvertently

Comment (dpansheriya) [reply migration/i7.1z/oracle/201602161510_IDEMPIERE-2955.sql]: @globalqss I just fixed that deleted comment mark.  I identified that but waiting for oracle setup done. Please test now. still I have not oracle DB ready but as you have tested, let's move this ticket.

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Pull request merged**